run/markdown-preview/editor/templates/markdown.md

```diff
@@ -0,0 +1,23 @@
+# Playing with Markdown
+
+This UI allows a user to write Markdown text and preview the rendered HTML.
+
+You may be familiar with this workflow from sites such as Github or Wikipedia.
+
+In practice, this web page does the following:
+
+* On click of the *"Preview Rendered Markdown"* button, browser JavaScript
+  lifts the markdown text and sends it to the editor UI's public backend.
+* The editor backend sends the text to a private Renderer service which
+  converts it to HTML.
+* The HTML is injected into the web page in the right-side **Rendered HTML** area.
+
+## Markdown Background
+
+From **[John Gruber on Daring Fireball](https://daringfireball.net/projects/markdown/)**:
+
+> "Markdown is a text-to-HTML conversion tool for web writers. Markdown allows
```

Comment:
Not sure the rules on quoting blocks like this. Might not want to include.

Reply:
Haven't really seen rules on this, visually I expect quotation characters or quotation styling to make the nature of the text clear. Since I do not have blockquote styling in the page this seems reasonable. Is it worth adding the necessary CSS?

Reply:
Confirmed to avoid. Removed.